src/ObservableEither.ts

```diff
@@ -0,0 +1,185 @@
+/**
+ * @since 0.6.8
+ */
+import * as E from 'fp-ts/lib/Either'
+import * as R from './Observable'
+import * as TE from 'fp-ts/lib/TaskEither'
+import { Bifunctor2 } from 'fp-ts/lib/Bifunctor'
+import { IO } from 'fp-ts/lib/IO'
+import { IOEither } from 'fp-ts/lib/IOEither'
+import { Monad2 } from 'fp-ts/lib/Monad'
+import { MonadObservable2 } from './MonadObservable'
+import { Observable, defer } from 'rxjs'
+import { Task } from 'fp-ts/lib/Task'
+import { getEitherM } from 'fp-ts/lib/EitherT'
+import { pipeable } from 'fp-ts/lib/pipeable'
+
+const T = getEitherM(R.observable)
+
+declare module 'fp-ts/lib/HKT' {
+  interface URItoKind2<E, A> {
+    ObservableEither: ObservableEither<E, A>
+  }
+}
+
+/**
+ * @since 0.6.8
+ */
+export const URI = 'ObservableEither'
+/**
+ * @since 0.6.8
+ */
+export type URI = typeof URI
+
+/**
+ * @since 0.6.8
+ */
+export interface ObservableEither<E, A> extends Observable<E.Either<E, A>> {}
+
+/**
+ * @since 0.6.8
+ */
+export const left: <E = never, A = never>(e: E) => ObservableEither<E, A> = T.left
+
+/**
+ * @since 0.6.8
+ */
+export const right: <E = never, A = never>(a: A) => ObservableEither<E, A> = T.of
+
+/**
+ * @since 0.6.8
+ */
+export const rightObservable: <E = never, A = never>(ma: Observable<A>) => ObservableEither<E, A> = T.rightM
+
+/**
+ * @since 0.6.8
+ */
+export const leftObservable: <E = never, A = never>(ma: Observable<E>) => ObservableEither<E, A> = T.leftM
+
+/**
+ * @since 0.6.8
+ */
+export const fromIOEither: <E, A>(fa: IOEither<E, A>) => ObservableEither<E, A> = R.fromIO
+
+/**
+ * @since 0.6.8
+ */
+export function rightIO<E, A>(ma: IO<A>): ObservableEither<E, A> {
+  return rightObservable(R.fromIO(ma))
+}
+
+/**
+ * @since 0.6.8
+ */
+export function leftIO<E, A>(me: IO<E>): ObservableEither<E, A> {
+  return leftObservable(R.fromIO(me))
+}
+
+/**
+ * @since 0.6.8
+ */
+export function fromTaskEither<E, A>(t: TE.TaskEither<E, A>): ObservableEither<E, A> {
+  return defer(t)
```

Comment:
You can use `fromTask` from `Observable`

Reply:
Yeah!